articles/users/search.md

```diff
@@ -27,6 +28,25 @@ In this document, we use the terms **eventually consistent** and **immediately c
 
 * **Immediately consistent**: When you request information about a user (or a group of users), the response will reflect the results of all successful write operations, including those that occured shortly prior to your request.
 
+## General Principles
+
+When running user searches:
+
+* Use an immediately consistent endpoint during authentication pipeline
```

Comment:
I don't get this sentence. What's `during authentication pipeline`? You probably mean something like authentication transaction?